src/plugins/intel_gpu/src/graph/impls/onednn/convolution_onednn.cpp

```diff
@@ -120,7 +110,8 @@ struct convolution_onednn : typed_primitive_onednn_impl<convolution> {
 
     static std::shared_ptr<dnnl::primitive_attr> get_primitive_attributes(const typed_program_node<convolution>& arg,
                                                                             const kernel_impl_params& impl_params,
-                                                                            int& zero_point_mask) {
+                                                                            int& zero_point_mask,
+                                                                            dnnl::memory::data_type& wzp_data_type) {
```

Comment:
`wzp_data_type` is unused. Did you intend to set it from here?

Reply:
It is used at create() function for binary serialization.